library/std/src/net/ip/tests.rs

```diff
@@ -679,6 +695,12 @@ fn ipv6_properties() {
         documentation
     );
 
+    check!(
+        "2001:2::ac32:23ff:21",
+        &[0x20, 1, 0, 2, 0, 0, 0, 0, 0, 0, 0xac, 0x32, 0x23, 0xff, 0, 0x21],
+        global | unicast_global | benchmarking
```

Comment:
Same as above, benchmarking addresses should not actually be globally reachable. They do however have global unicast scope.